android/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@mipmap/ic_launcher_background"/>
+    <background android:drawable="@color/ic_launcher_background"/>
```

Comment:
can we delete all the ic_launcher_background.png files? Which seem to be the Valora icon bg?

Reply:
The new icons use this background color. Without these lines, the icons will be rendered with a black background instead of blue.

Reply:
I meant the @mipmap/ic_launcher_background.png and other similar files, which are currently on main and have the valora icon, it seems like we've replaced the background image with the color, so is there a reason to still keep those files around?

Reply:
Removed in c6de3d0c700de8b108362b3e2f7c3680c012b0b8. Also found some older Celo Icons that are now replaced.